eZ/Bundle/EzPublishCoreBundle/Matcher/MatcherServiceRegistry.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Matcher;
+
+use eZ\Publish\Core\Base\Exceptions\NotFoundException;
+use eZ\Publish\Core\MVC\Symfony\Matcher\MatcherInterface;
+
+class MatcherServiceRegistry
+{
+    /** @var \eZ\Publish\Core\MVC\Symfony\Matcher\MatcherInterface[] */
+    private $matchers;
+
+    /**
+     * MatcherServiceRegistry constructor.
+     *
+     * @param \eZ\Publish\Core\MVC\Symfony\Matcher\MatcherInterface[] $matchers
+     */
+    public function __construct(array $matchers = [])
+    {
+        $this->matchers = $matchers;
+    }
+
+    /**
+     * @param \eZ\Publish\Core\MVC\Symfony\Matcher\MatcherInterface[] $matchers
+     */
+    public function setMatchers(array $matchers): void
```

Comment:
Methods `\eZ\Bundle\EzPublishCoreBundle\Matcher\MatcherServiceRegistry::{setMatchers, getMatchers, hasMatcher, setMatcher}` seems to be unused.